openprescribing/measure_definitions/cgm_sensors.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "Prescribing of continuous glucose monitoring sensors",
+  "title": "",
+  "description": "Prescribing of continuous glucose monitoring sensors per 1000 patients",
+  "numerator_short": "Continuous glucose monitoring sensor quantity",
+  "denominator_short": "1000 patients",
+  "why_it_matters": [
+    "<p>Continuous glucose monitoring devices (CGM) can significantly reduce the need for people with diabetes to conduct finger-prick testing.</p>",
```

Remove leading paragraph tag to prevent new line at the start for consistency.
```suggestion
    "Continuous glucose monitoring devices (CGM) can significantly reduce the need for people with diabetes to conduct finger-prick testing.</p>",
```